components/Button.js

```diff
@@ -1,9 +1,13 @@
 import { StyledButtonOrLink } from "./StyledButtonOrLink";
 import { Icon } from "@iconify/react";
 
-function Button({ onClick, buttonName, buttonIcon, $backgroundColor }) {
+function Button({ onClick, buttonName, buttonIcon, $backgroundColor, type }) {
   return (
-    <StyledButtonOrLink onClick={onClick} $backgroundColor={$backgroundColor}>
+    <StyledButtonOrLink
+      onClick={onClick}
+      $backgroundColor={$backgroundColor}
+      type={type}
+    >
```

Comment:
```suggestion
    <StyledButtonOrLink
      onClick={onClick}
      $backgroundColor={$backgroundColor}
      {...props}
    >
```

unusual to pass native html attributes with their names, normally you would just pass the rest of all the other props